### PR TITLE
fix(API): assign contribution_id is of type ContributionId

### PR DESCRIPTION
**Pull request by tdelabro**

### CODE REVIEW

File: src/dto/u256.rs [hunk 1]
```diff
@@ -0,0 +1,59 @@
+use crypto_bigint::{Encoding, U256};
+use thiserror::Error;
+
+use super::{hex_prefixed_from_str, ParseHexPrefixedStringError};
+
+#[derive(Debug, Error)]
+pub enum ParseU256Error {
+	#[error("provided sting shoud be between 3 and 66 characters long")]
+	InvalidLenght,
```
Comment (tekkac) [inline src/dto/u256.rs:9]:
```suggestion
	InvalidLength,
```

File: src/routes/dto/u256/tests.rs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+use super::*;
+use serde_test::{assert_de_tokens, assert_de_tokens_error, Token};
+
+#[test]
+fn deserialize() {
+	assert_de_tokens(
+		&U256Param(U256::from_u128(81985529216486895)),
+		&[Token::Str("0x0123456789abcdef")],
```
Comment (tekkac) [inline src/routes/dto/u256/tests.rs:8]: Does this mean we don't need to pad zeros to `contributor_id` anymore?

**Review by tekkac — APPROVED** [review]
that was a long read 
Lgtm

File: src/routes/dto/u256/tests.rs (hunk 2 above)
Comment (tdelabro) [reply src/routes/dto/u256/tests.rs:8]: yup id does. Took me a while to implement correctly but we are fine now

File: src/dto/u256.rs (hunk 1 above)
Comment (tdelabro) [reply src/dto/u256.rs]: done

**Pull request merged**